docs/section11/Rule11-13.md

```diff
@@ -0,0 +1,49 @@
+# Service_Water_Heating - Rule 11-13  
+**Schema Version:** 0.0.37  
+
+**Mandatory Rule:** FALSE  
+
+**Rule ID:** 11-13  
+
+**Rule Description:** "Service water-heating energy consumption shall be calculated explicitly based upon the volume of service water heating required and the entering makeup water and the leaving service water-heating temperatures. Entering water temperatures shall be estimated based upon the location. Leaving temperatures shall be based upon the end-use requirements."  
+
+**Rule Assertion:** UNDETERMINED / NOT_APPLICABLE  
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, (e)  
+
+**Evaluation Context:** B_RMD - each building  
+**Data Lookup:**   
+**Function Call:**  
+- **get_SWH_uses_associated_with_each_building_segment**
+- **get_energy_required_to_heat_swh_use**  
+- **get_component_by_id**  
+
+## Applicability Checks:  
+- look at each building in the RMD: `for building in B_RMD:`
+    - create a dictionary for this building that holds information about the service water heating uses: `service_water_heating_info = {"BTU_PER_YEAR": 0, "BTU_PER_SF_PER_YEAR":0}
+    - create a boolean is_applicable and set it to false: `is_applicable = false`
+    - look at each building segment in the building: `for building_segment in building.building_segments:`
+        - get the service water heating uses in the building segment `service_water_heating_use_ids = get_SWH_uses_associated_with_each_building_segment(P_RMD, building_segment.id)`
+        - look at each service water heating use id: `for swh_use_id in service_water_heating_use_ids:`
+            - get the swh_use using get_component_by_ID: `swh_use = get_component_by_ID(P_RMD, swh_use_id)`
+            - check to see if the use has SWH loads: `if swh_use.use > 0:`
+                - the rule is applicable: `is_applicable = true`
+                - now get the energy required to heat this swh use using the function get_energy_required_to_heat_swh_use: `energy_required_by_space_id = get_energy_required_to_heat_swh_use(swh_use, B_RMD, building_segment)`
+                - if any of the energy required is UNDETERMINED, we need to make BTU_PER_YEAR UNDETERMINED: `if "UNDETERMINED" in energy_required_by_space_id.values():`
+                    - set BTU_PER_YEAR to UNDETERMINED: ` service_water_heating_info["BTU_PER_YEAR"] = "UNDETERMINED"`
+                    - set BTU_PER_SF_PER_YEAR to UNDETERMINED: ` service_water_heating_info["BTU_PER_SF_PER_YEAR"] = "UNDETERMINED"`
+                - otherwise, if BTU_PER_YEAR is not UNDETERMINED, add the sum of the values of energy_required_by_space_id to the total: `if service_water_heating_info["BTU_PER_SF_PER_YEAR"] != "UNDETERMINED":`
+                    - add the sum of the values of to the total: ` service_water_heating_info["BTU_PER_YEAR"] += sum(energy_required_by_space_id.values())`
```

Comment:
same comment as above (`energy_required_by_space_id` does not seem to be a dictionary)